dags/looker.py

```diff
@@ -153,4 +153,33 @@
         **airflow_gke_prod_kwargs,
     )
 
-    lookml_generator_staging >> lookml_generator_prod
+    looker_folders_to_validate = [
+        706,  # KPI metrics
+    ]
+
+    lookml_validate_spectacles = GKEPodOperator(
+        owner="ascholtz@mozilla.com",
+        email=[
+            "ascholtz@mozilla.com",
+            "telemetry-alerts@mozilla.com",
+        ],
+        task_id="lookml_validate_spectacles",
+        name="lookml-validate-spectacles",
+        image="gcr.io/moz-fx-data-airflow-prod-88e0/lookml-generator:latest",
+        dag=dag,
+        cmds=["bash", "-x", "-c"],
+        arguments=[
+            "pip install spectacles==2.4.10 &&" # todo: remove this once mozilla-nimbus-schemas supports newer pydantic version
```

Comment:
`lookml-generator` does have `spectacles` installed, but it needed to be pinned to version <=2.3.16 due to incompatible dependencies (pydantic) with mozilla-nimbus-schemas. These older versions however encounter some issue when connecting to the Looker API. Manually installing a more recent version here seems to work though for now.
I already pinged Mike to see if pydantic can be updated for mozilla-nimbus-schemas, but it looks like a more difficult thing to do.